ingest.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+
```

Comment:
SUGGESTION: add `set -euo pipefail` to have some error-handling

Reply:
will do